penrose-web/src/toCustomAD.ts

```diff
@@ -0,0 +1,367 @@
+import {API, ASTPath, BinaryExpression, CallExpression, Collection, FileInfo, Identifier, Node,
```

Comment:
Is this a duplicate of what's in `src/mathtransform` and `src/shapes/mathtransform`? There seemed to be 3 copies of the same file.

Reply:
Cleaned up in latest commit.